### PR TITLE
Feature/static call

### PR DESCRIPTION
**Pull request by jesusholoo**
- add static call tests

### CODE REVIEW

**Review by ignasirv — APPROVED** [review]
💯

**Review by laisolizq — APPROVED** [review]
:smile_cat:

File: tools/ethereum-tests/no-exec.json [hunk 1]
```diff
@@ -161,6 +170,22 @@
     {
       "name": "stMemoryTest/buffer_248",
       "description": "Triggers an out of zk-counters error"
+    },
+    {
+    "name": "StaticcallToPrecompile",
```
Comment (krlosMata) [inline tools/ethereum-tests/no-exec.json:175]: Please, setup the name in the following format: `#folder/#test-name`

**Review by krlosMata — CHANGES_REQUESTED** [review]

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Pull request merged**